app/mailers/acceptance.rb

```diff
@@ -0,0 +1,6 @@
+class Acceptance < ApplicationMailer
```

Comment:
Will do!  For some reason the guide i was following didn't have the word mailer in it so I thought it was standard convention but I just looked around and most other things have the word mailer with it.